### DISCUSSION TITLE
Set up notifications (Amazon EventBridge workflow) step 3

### DISCUSSION THREAD

**Discussion by ex-melody** [amzn/selling-partner-api-models#3075] [General]
https://developer-docs.amazon.com/sp-api/docs/notifications-api-v1-use-case-guide#step-3-create-a-rule-that-triggers-on-a-notification-event

6. In the Define pattern area, select Event pattern.

7. Select Pre-defined pattern by service.

where is Define pattern area?

![image](https://user-images.githubusercontent.com/62811728/195010299-dbe31c8b-cfd7-4a9c-a43b-ce7d274595eb.png)

**Comment by tomaszhanc** [comment]
@ex-melody iirc it is in the next step after clicking "NEXT" button. Did you try it?
